mmdet3d/models/dense_heads/mlp_bbox_head.py

```diff
@@ -0,0 +1,123 @@
+from mmcv.cnn import ConvModule
+from torch import nn as nn
+
+from mmdet.models.builder import HEADS
+
+
+@HEADS.register_module()
+class MLPBBoxHead(nn.Module):
+    r"""More general bbox head, with shared mlp layers and two optional
+    separated branches.
+
+    .. code-block:: none
+
+                   /-> cls mlps -> cls_score
+        shared mlps
+                   \-> reg mlps -> bbox_pred
+    """
+
+    def __init__(self,
+                 in_channels=0,
+                 shared_mlp_channels=(),
+                 cls_mlp_channels=(),
+                 num_cls_out_channels=0,
+                 reg_mlp_channels=(),
+                 num_reg_out_channels=0,
+                 conv_cfg=dict(type='Conv1d'),
```

Comment:
If it is Conv1d here, maybe we could not call it as mlpbboxhead. We may use `BaseAnchorFree3DHead`. And use it as a base for VoteHead & SSD3DHead.